### PR TITLE
Add show_cursor method

### PR DESCRIPTION
**Pull request by bmathews**
Exposes the SDL ShowCursor method.

Two things worth mentioning:

1) I wasn't sure if the 3 int values should be exposed as some enum kind of deal.
2) I have no clue how to build/test this. 😭 

I only started playing around with reason yesterday, so I've still got some learning to do.

### CODE REVIEW

File: src/tsdl_new.c [hunk 1]
```diff
@@ -408,3 +408,9 @@ CAMLprim value TSDL_GetWindowMaximumSize(value window) {
 
   CAMLreturn(ret);
 }
+
+CAMLprim value TSDL_ShowCursor(value toggle) {
+  CAMLparam1(toggle);
+  int visible = SDL_ShowCursor(toggle);
```
Comment (bsansouci) [inline src/tsdl_new.c]: You need to call the macro Int_val on toggle before it’s actually an int, otherwise from C’s perspective it’s wrong.

**Review by bsansouci — COMMENTED** [review]
Hi! Thanks for this, looks great. 
Can you add the three constants that would be passed to this? 
SDL_ENABLE, SDL_DISABLE, SDL_QUERY

**Comment by bsansouci** [comment]
The simplest way to test this would be to install a example repo, like github.com/bsansouci/reprocessing-example and symlink your local clone of tsdl inside the node_modules, then clean and build the example project (make sure to clean, otherwise you might get stale artifacts, because editing your node_modules isn’t really a supported thing)

**Comment by bmathews** [comment]
Any thoughts on where these should go or how they should be exposed?

```
module Sdl = {
  let query = -1;
  let disable = 0;
  let ignore = 0;
  let enable = 1;
}
```

These same constants are reused by a handful of other SDL methods, so it probably shouldn't be named specific to the cursor values?

**Comment by bsansouci** [comment]
Can you do what we did for tgls and simply prefix them `sdl_enable` and put them at the bottom of the file (before the big comments which should be removed)

**Comment by bmathews** [comment]
It works! should be good to go now!

**Comment by bmathews** [comment]
Fixing conflicts now. 

Edit: ✅

**Comment by bsansouci** [comment]
Great, thanks a lot!

**Pull request merged**